easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -177,6 +186,8 @@ function create_project(){
 }
 
 function activate_env(){
+    prepend_miniconda
```

Comment:
This means that easy-deploy usages of our scripts are now pegged to miniconda's python, not the system python, right? That's probably a good thing.